ext/config.m4

```diff
@@ -217,6 +217,10 @@ if test "$PHP_CASSANDRA" != "no"; then
     done
   fi
 
+#changed by mohamad zeynali mohammad.basu@gmail.com (apache license) to fix undefined reference to `deflateInit_' bug
```

Comment:
Thanks for the regards. Would you like me to amend df8d7b2 commit or just create another commit with the mentioned comment? I just created a comment on the commit.